src/www/ui_components/app-root.js

```diff
@@ -271,7 +271,7 @@ export class AppRoot extends mixinBehaviors([AppLocalizeBehavior], PolymerElemen
         </app-header>
 
         <iron-pages id="pages" selected="[[page]]" attr-for-selected="name">
-          <servers-view name="servers" id="serversView" servers="[[servers]]" localize="[[localize]]"></servers-view>
+          <servers-view name="servers" id="serversView" servers="[[servers]]" localize="[[localize]]" language="[[language]]" platform="[[platform]]"></servers-view>
```

Comment:
Set the message selection here with a computed value that depends on language.

Reply:
Done